loader/index.js

```diff
@@ -0,0 +1,14 @@
+import wasm from '../build/optimized.wasm';
+
+let modules;
+
+wasm({ ...imports }).then(instance => {
```

Comment:
This is temporary, before I implement a better solution for exporting WebAssembly modules. Solutions are either:

- Complying on @assemblyscript/loader using `loader.instantiateStreaming`
- Making a WASM file for each module (curently max size for synchronous compilation is 4KB). Current WASM file is >100KB with `-O3` optimization
- Develop a processor script to pass `<script>` to WebAssembly using `eval` and `<noscript>`

Reply:
### [WebAssembly.Module.exports()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Module)

Reply:
I'm currently experimenting with [as-bind](https://github.com/torch2424/as-bind)